frontend/src/game/scenes/Collect.ts

```diff
@@ -52,6 +64,25 @@ export class Collect extends Scene {
   }
 
   update(time: number, delta: number) {
+    this.sweeper.setVisible(false);
```

Comment:
This can be moved into an `else` under `if (this.input.activePointer.isDown)` so it doesn't get run every time